lib/kafka/broker_pool.rb

```diff
@@ -2,12 +2,14 @@
 
 module Kafka
   class BrokerPool
-    def initialize(client_id:, connect_timeout: nil, socket_timeout: nil, logger:)
+    def initialize(client_id:, connect_timeout: nil, socket_timeout: nil, logger:, ssl: nil, ssl_context: nil)
```

Comment:
ok. I started to write that, then realized I didn't know if you wanted to do it. I will attach it to this PR now.